Client/Frontend/Browser/BrowserViewController/BVC+Rewards.swift

```diff
@@ -304,13 +304,9 @@ extension Tab {
     }
 
     group.notify(queue: .main) {
-      let faviconURL = URL(string: self.displayFavicon?.url ?? "")
-      if faviconURL == nil {
-        log.warning("No favicon found in \(self) to report to rewards panel")
-      }
       rewards.reportLoadedPage(
         url: url, redirectionURLs: urls.isEmpty ? [url] : urls,
-        faviconURL: faviconURL, tabId: Int(self.rewardsId),
+        faviconURL: nil, tabId: Int(self.rewardsId),
```

Comment:
Because we don't have a `faviconURL`. The URL for the favicon is pulled from Brave-Core now.

Reply:
We may need to check if this affects the rewards panel though because we show a favicon there for the publisher... Please visit `wikipedia.org` and ensure it displays the favicon like so:
<img width="377" alt="image" src="https://user-images.githubusercontent.com/529104/178801992-d81c695e-bc32-4016-a4f8-2fff6b1d6c3c.png">

Reply:
It works correctly in the rewards panel. This is just for some reporting thing.

Reply:
I think this will probably break some underlying code in the rewards system but it may not matter in this case anymore since we don't show any of the details